client/my-sites/plan-features/index.jsx

```diff
@@ -425,6 +467,20 @@ class PlanFeatures extends Component {
 		this.props.recordTracksEvent( 'calypso_wp_plans_test_view' );
 		retargetViewPlans();
 	}
+
+	componentDidMount() {
+		const {
+			isInSignupTest,
+			displayJetpackPlans
+		} = this.props;
+
+		// center plans
+		if ( isInSignupTest ) {
+			displayJetpackPlans
+				? document.querySelector( '.jetpack-connect__plans .plans-wrapper' ).scrollLeft = 150
```

Comment:
I don't think this is a React way to scroll an element. You should not directly refer DOM elements using any DOM selector functions such as `querySelector` and `getElementById`.